## Welcome to my personal page

I am currently a Senior Bioinformatics Research Officer at LeedsOmics, University of Leeds. My primary responsibility is to maintain the day-to-day running of LeedsOmics Institute, organise and coordinate LeedsOmics activities such as annual symposiums, research seminar series, training workshops and coding clubs as well as provide the data analysis and training for the researchers in terms of bioinformatics and other cutting-edge omics technologies.

I hold a B.Sc degree in the mathematics from Shandong University (2006) and a Ph.D degree in genomics and bioinformatics from Beijing Institute of Genomics (BIG), Chinese Academy of Science (2011). After graduation, I was appointed as Assistant Professor and Associate Professor in the same institute (2011-2014). During the period in Beijing, I was focusing on the study of genome evolution by means of a large set of genomes from myriad species that span a diversity of taxonomies.

Afterwards, I moved to UK and accepted an appointment as a Research Associate in Cancer Institute, University College London for nearly two years (2014-2016) where I endeavored to construct several decent comprehensive pipelines for RNA-Seq and ChIP-Seq analyses with their applications on cancer research such as Leukemia study.

After leaving London, I spent two years at Oxford (2016-2018) as a Postdoctoral Research Assistant at Department of Plant Sciences, University of Oxford where I was working on conducting RNA-Seq analysis to study the mechasims of various Conifer species in response to the diverse abiotic and biotic changes in terms of gene expression profiles and sequence variations.

# Employment history
Feb. 2018-Present, Senior Bioinformatics Research Officer, LeedsOmics, University of Leeds, Leeds, UK
Feb. 2016-Jan. 2018, Postdoctoral Research Assistant, Department of Plant Sciences, University of Oxford, Oxford, UK (With Prof. John Mackay)
Mar. 2014-Jan. 2016, Research Associate, UCL Cancer Institute, University College London, London, UK (With Prof. Enver Tariq and Dr. Javier Herrero)
Jan. 2014-Feb. 2014, Associate Professor, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China
Jul. 2011-Dec. 2013, Assistant Professor, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China

# Education
Sep. 2006-Jul. 2011, PhD, Genomics and Bioinformatics, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China (With Prof. Jun Yu)
Sep. 2002-Jul. 2006, BS, Information and Computing Science, School of Mathematics and System Sciences, Shandong University, Jinan, China

# Publication
FIRST/CO-FIRST AUTHOR(#) AND/OR CORRESPONDING/CO-CORRESPONDING AUTHOR(*)
Under review or in preparation
25. Dapeng Wang*. ChIPwrapper: A highly-efficient wrapper for handling massive ChIP-Seq samples. In preparation.

24. Dapeng Wang*. denovo-hppRNA: A smart pipeline for RNA-Seq analysis from reference-free genomes. In preparation.

23. Dapeng Wang*. GCevobase: an evolution-based database for GC content in eukaryotic genomes. Under review.

2017
22. Charlotta Böiers, Simon E. Richardson, Emma Laycock, Alya Zriwil, Virginia A. Turati, John Brown, Jason P. Wray, Dapeng Wang, Chela James, Javier Herrero, Ewa Sitnicka, Stefan Karlsson, Andrew J.H. Smith, Sten Erik W. Jacobsen and Tariq Enver. A Human IPS Model Implicates Embryonic B-Myeloid Fate Restriction as Developmental Susceptibility to B Acute Lymphoblastic Leukemia-Associated ETV6-RUNX1. Developmental Cell. 2017 Dec 27. pii: S1534-5807(17)30990-5. PMID: 29290585

21. Itaru Kato, Yoko Nishinaka, Masahiro Nakamura, Ayse U. Akarca, Akira Niwa, Hiroki Ozawa, Kenichi Yoshida, Makiko Mori, Dapeng Wang, Makiko Morita, Hiroo Ueno, Yusuke Shiozawa, Yuichi Shiraishi, Satoru Miyano, Rajeev Gupta, Katsutsugu Umeda, Kenichiro Watanabe, Katsuyoshi Koh, Souichi Adachi, Toshio Heike, Megumu K. Saito, Masashi Sanada, Seishi Ogawa, Teresa Marafioti, Akira Watanabe, Tatsutoshi Nakahata and Tariq Enver. Hypoxic adaptation of leukemic cells infiltrating the CNS affords a therapeutic strategy targeting VEGFA. Blood. 2017 Jun 8;129(23):3126-3129. PMID: 28424164

20. Dapeng Wang*. hppRNA – A Snakemake-based handy parameter-free pipeline for RNA-Seq analysis of numerous samples. Briefings In Bioinformatics. 2017 Jan 17. pii: bbw143. PMID:28096075

2016
19. Dapeng Wang*. DLGP: A database for lineage-conserved and lineage-specific gene pairs in animal and plant genomes. Biochemical and Biophysical Research Communications 469 (2016) 542-545. PMID: 26697753

2015
18. Dapeng Wang*, Jiayue Xu and Jun Yu. KGCAK: A K-mer based database for genome-wide phylogeny and complexity evaluation. Biology Direct 2015, 10:53. PMID: 26376976

17. Dapeng Wang* and Jun Yu. LCGserver: a webserver for exploring evolutionary trajectory of gene orders in a large number of genomes. OMICS: A Journal of Integrative Biology. 2015 Sep;19(9):574-7. PMID: 26258441

16. Dapeng Wang* and Jun Yu. Plastid-LCGbase: a collection of evolutionarily conserved plastid-associated gene pairs. Nucleic Acids Research. 2015 Jan;43(Database issue):D990-5. PMID: 25378306

15. Cristina Pina, Jose Teles, Cristina Fugazza, Gill May, Dapeng Wang, Yanping Guo, Shamit Soneji, John Brown, Patrik Eden, Mattias Ohlsson, Carsten Peterson and Tariq Enver. Single-cell network analysis identifies ddit3 as a nodal lineage regulator in hematopoiesis. Cell Reports 2015 Jun 16;11(10):1503-10. PMID: 26051941

2013
14. Bin Xie, Dapeng Wang#, Yong Duan, Jun Yu, and Hongxing Lei. Functional networking of human divergently paired genes. PLoS ONE. 2013, 8(10): e78896. PMID: 24205343

13. Yunbo Li, Dapeng Wang#, Lei Wang, Jinhai Yu, Danhua Du, Ye Chen, Peng Gao, Duen-Mei Wang, Jun Yu, Feng Zhang, and Shuanglin Fu. Distinct genomic aberrations between low-grade and high-grade gliomas of Chinese patients. PLoS ONE. 2013, 8(2): e57168. PMID: 23451178 Highlighted in GenomeWeb Daily Scan

12. Dapeng Wang, Yan Xia, Xinna Li, Lixia Hou and Jun Yu. The Rice Genome Knowledgebase (RGKbase): an annotation database for rice comparative genomics and evolutionary biology. Nucleic Acids Research. 2013; 41: D1199-D1205.PMID: 23193278

11. Lei Wang, Xiaowei Zhang, Linlin Pan, Wanfei Liu, Dapeng Wang, Guangyu Zhang, Yuxin Yin, An Yin, ShangangJia, Xiaoguang Yu, Gaoyuan Sun, Songnian Hu, Ibrahim S. Al-Mssallem, and Jun Yu. A Large-scale gene discovery for the red palm weevil Rhynchophorus ferrugineus (Coleoptera: Curculionidae). Insect Science 2013, 20: 689–702. PMID: 23955844

2012
10. Dapeng Wang, Yao Su, Xumin Wang, Hongxing Lei and Jun Yu. Transposon-Derived and Satellite-Derived Repetitive Sequences Play Distinct Functional Roles in Mammalian Intron Size Expansion. Evolutionary Bioinformatics 2012:8 301-319. PMID: 22807622

9. Dapeng Wang, Yubin Zhang, Zhonghua Fan, Guiming Liu and Jun Yu. LCGbase: A Comprehensive Database for Lineage-Based Co-regulated Genes. Evolutionary Bioinformatics 2012:8 39-46. PMID: 22267903

2011
8. Dapeng Wang and Jun Yu. Both size and GC-content of minimal introns are selected in human populations. PLoS ONE 2011, 6(3): e17945. PMID: 21437290 Highlighted in GenomeWeb Daily Scan

7. Dapeng Wang, Fei Liu, Lei Wang, Shi Huang and Jun Yu. Nonsynonymous substitution rate (Ka) is a relatively consistent parameter for defining fast-evolving and slow-evolving protein-coding genes. Biology Direct 2011, 6:13. PMID: 21342519

6. Yan-Hui Zhao, Da-Peng Wang#, Li-Li Zhang, Feng Zhang, Duen-Mei Wang and Wei-Yuan Zhang. Genomic expression profiles of blood and placenta reveal significant immune-related pathways and categories in Chinese women with gestational diabetes mellitus. Diabetic Medicine 2011, 28: 237–246. PMID: 21219437

2010
5. Feng Liu, Dapeng Wang#, Jianjun Fu, Gaoyuan Sun, Yubang Shen, Lingli Dong, Bing Zhang, Songnian Hu, and Jiale Li. Identification of immune-relevant genes by expressed sequence tag analysis of head kidney from grass carp (Ctenopharyngodon idella). Comparative Biochemistry and Physiology Part D: Genomics and Proteomics 2010, 5(2):116-23. PMID: 20403773

4. Dapeng Wang, Yubin Zhang, Zhang Zhang, Jiang Zhu, and Jun Yu. KaKs Calculator 2.0: a toolkit of incorperating gamma methods and sliding window strategies. Genomics Proteomics & Bioinformatics 2010, 8(1): 77-80. PMID: 20451164

3. Jiang Zhu, Fuhong He, Dapeng Wang, Kan Liu, Dawei Huang, Jingfa Xiao, Jiayan Wu, Songnian Hu, and Jun Yu. A novel role for minimal introns: routing mRNAs to the cytosol. PLoS ONE 2010, 5(4): e10144. PMID: 20419085

2009
2. Dapeng Wang, Song Zhang, Fuhong He, Jiang Zhu, Songnian Hu, and Jun Yu. How do variable substitution rates influence the calculations of Ka and Ks? Genomics Proteomics & Bioinformatics 2009, 7(3): 116-127. PMID: 19944384

1. Da-Peng Wang, Hao-Lei Wan, Song Zhang, and Jun Yu. Gamma-MYN: a new algorithm for estimating Ka and Ks with consideration of variable substitution rates. Biology Direct 2009, 4:20. PMID: 19531225


You can use the [editor on GitHub](https://github.com/NextGenBioinformatics/dapengwang/edit/master/README.md) to maintain and preview the content for your website in Markdown files.

Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.

### Markdown

Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for

```markdown
Syntax highlighted code block

# Header 1
## Header 2
### Header 3

- Bulleted
- List

1. Numbered
2. List

**Bold** and _Italic_ and `Code` text

[Link](url) and ![Image](src)
```

For more details see [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).

### Jekyll Themes

Your Pages site will use the layout and styles from the Jekyll theme you have selected in your [repository settings](https://github.com/NextGenBioinformatics/dapengwang/settings). The name of this theme is saved in the Jekyll `_config.yml` configuration file.

### Support or Contact

Having trouble with Pages? Check out our [documentation](https://help.github.com/categories/github-pages-basics/) or [contact support](https://github.com/contact) and we’ll help you sort it out.
